### ISSUE TITLE
Readme does not render the flow charts in the Github mobile app

### ISSUE THREAD

**Issue by jpc0** [amaargiru/pyroad#2]
In the github mobile app the readme does not render the flowcharts, only getting code dumps.

**Comment by jpc0** [comment]
![image](https://user-images.githubusercontent.com/12384344/203810501-9cf42f64-5aee-45c5-a610-af482ea1abf7.png)

Image for reference

**Comment by amaargiru** [comment]
Yes, looks like GitHub have some sort of problem with rendering Mermaid. I will replace the Mermaid-diagrams with png-images.
